vunit/sim_if/ghdl.py

```diff
@@ -347,8 +367,16 @@ def simulate(  # pylint: disable=too-many-locals
         )
 
         status = True
+
+        gcov_env = environ.copy()
```

Comment:
What's the motivation to copy the environment here instead of setting `environ["GCOV_PREFIX"]`? Is it because of parallel executions?

Reply:
Yes, it's because of parallel executions. GCOV_PREFIX needs different values for different tests, which is why it is set per subprocess spawned.